### PR TITLE
Add initial tests

### PR DESCRIPTION
**Pull request by bmwwoo**

### CODE REVIEW

**Comment by ericthewry** [comment]
@sf-interns/owners 
Hey guys please take a look at these. Let me know if there's anything I can do.

File: app/InputSolver.coffee [hunk 1]
```diff
@@ -7,23 +7,20 @@ class InputSolver
...
+    if op is "+" then return sum + parseInt(element)
+    if op is "-" then return sum - parseInt(element)
+    if op is "*" then return sum * parseInt(element)
     return sum
 
-  @compute = (input) ->
-    input = @parseInput input
-    previous = input[0]
-    sum = parseInt input[0]
+  @compute =  (input) ->
```
Comment (bmwwoo) [inline app/InputSolver.coffee]: extra space before input

File: app/InputSolver.coffee [hunk 2]
```diff
@@ -7,23 +7,20 @@ class InputSolver
     return element is "+" or element is "-" or element is "*"
 
   @operation = (sum, element, op) ->
-    if op is "+"
-      sum = sum + parseInt(element)
-    else if op is "-"
-      sum = sum - parseInt(element)
-    else if op is "*"
-      sum = sum * parseInt(element)
+    if op is "+" then return sum + parseInt(element)
```
Comment (bmwwoo) [inline app/InputSolver.coffee]: if we're doing single quotes as our standard, we should change these

**Comment by bmwwoo** [comment]
We'll just add a few more tests such as two operators in a row `+-` and a few minor elegance changes then LGTM :+1:

**Comment by ericthewry** [comment]
@bmwwoo If there are two operators in a row we want it to evaluate only the first?

**Comment by ericthewry** [comment]
Feel free to take a look when you get the chance

**Pull request merged**